crates/plugins/mediator-coordination/src/didcomm/bridge.rs

```diff
@@ -36,25 +37,37 @@ impl DIDResolver for LocalDIDResolver {
             return Ok(Some(self.diddoc.clone()));
         }
 
-        if !did.starts_with("did:key:") {
+        if did.starts_with("did:peer:") {
+            // Add logic to resolve Peer DIDs here
+            let method = DidPeer::new_with_format(PublicKeyFormat::Jwk); // Assuming a Peer DID resolver
+            match method.expand(did) {
+                Ok(diddoc) => Ok(Some(
+                    serde_json::from_value(json!(Document {
+                        service: Some(vec![]),
+                        ..diddoc
+                    }))
+                    .expect("Should easily convert between DID document representations."),
```

Comment:
Find a better way to handle the error without panicking.

Reply:
There's no need to propagate errors from this conversion, cause it is a hack; and it is not expected to fail.

Reply:
Ok. We can simply `unwrap` then if it will never fail. Yes yes `expect` add more runtime code and space than unwrap. mdr